scripts/sort/index.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+const { join } = require('path')
+const sortJSON = require('./sortJSON')
+const sortYamlFile = require('./sortYamlFile')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/omrilotan/isbot/pullRequest?prid=7728710)
```suggestion
const sortYamlFile = require("./sortYamlFile")
```